.devcontainer/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN apt-get update \
     && DEBIAN_FRONTEND="noninteractive" apt-get install --yes \
       bash \
       build-essential \
-      containerd.io=1.5.11-1 \
+      containerd.io \
```

Comment:
Let's remove the above comment about pinning `containerd.io`, as it would no longer apply.